counterpartylib/test/fixtures/vectors.py

```diff
@@ -1498,6 +1505,11 @@
                     'quantity': 9223372036854775807,
                 }}
             ]
+        }, {
+            'comment': 'Reject a send without memo to a REQUIRE_MEMO address',
+            'mock_protocol_changes': {'options_require_memo': True},
+            'in': ({'block_index': DP['default_block_index'], 'block_time': 155409000, 'fee': 10000, 'tx_index': 502, 'tx_hash': '8fc698cf1fcd51e3d685511185c67c0a73e7b72954c6abbd29fbbbe560e043a0', 'btc_amount': 7800, 'data': b'\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x00\x02\xfa\xf0\x80', 'source': ADDR[0], 'destination': ADDR[6], 'supported': 1, 'block_hash': DP['default_block_hash']},),
+            'out': None
```

Comment:
Close on this one.  Check the sends table to find the invalid record.

See a similar example in the test with comment `instead of auto-correcting the quantity to the amount the address holds return invalid: insufficient funds`